pkg/reconciler/configuration/resources/revision.go

```diff
@@ -59,6 +60,7 @@ func UpdateRevisionLabels(rev, config metav1.Object) {
 		serving.ConfigurationLabelKey,
 		serving.ServiceLabelKey,
 		serving.ConfigurationGenerationLabelKey,
+		networking.IngressClassAnnotationKey,
```

Comment:
I can see someone coming by here confused why the annotation key is in the labels. 

We should probably call this `IngressClassLabelKey` even if it is the same key, or at least add a comment that this is here so that labelSelectors work on this.

Reply:
Done